src/index.html

```diff
@@ -1,13 +1,494 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>MyBike</title>
+    <link
+      rel="shortcut icon"
+      type="image/png"
+      href="./images/favicon-b.png"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
+    <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container container__header">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="/" class="logo">
```

Comment:
add # instead of / because i have an error when clicking on the logo(in menu do the same)